### PR TITLE
Add 3.0-RC2 schemas

### PR DESCRIPTION
**Pull request by tdelmas**
Include RC2 changes (listed in the commits names)

v3.0-RC2 release note: https://github.com/MobilityData/gbfs/releases/tag/v3.0-RC2

### CODE REVIEW

**Comment by richfab** [comment]
Thank you @tdelmas for this very helpful contribution! ⚡ 

I would recommend to include https://github.com/MobilityData/gbfs/pull/559 in this PR once the vote has passed (the vote closes in a few hours).

File: v3.0-RC2/vehicle_status.json [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+                    "format": "uri"
+                  }
+                }
+              },
+              "vehicle_type_id": {
+                "description": "The vehicle_type_id of this vehicle (added in v2.1-RC).",
+                "type": "string"
+              },
+              "last_reported": {
+                "description": "The last time this vehicle reported its status to the operator's backend in POSIX time (added in v2.1-RC).",
```
Comment (richfab) [inline v3.0-RC2/vehicle_status.json:91]: Let's please update the description to [RFC3339 format](https://www.rfc-editor.org/rfc/rfc3339).

Thank you!

```suggestion
                "description": "The last time this vehicle reported its status to the operator's backend in RFC3339 format (added in v2.1-RC).",
```

File: v3.0-RC2/system_alerts.json [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+                    }
+                  },
+                  "required": ["text", "language"]
+                }
+              },
+              "last_updated": {
+                "description":
+                  "Indicates the last time the info for the alert was updated.",
+                "type": "number",
+                "minimum": 1450155600
```
Comment (richfab) [inline v3.0-RC2/system_alerts.json:154]: Let's please update the type and format to [RFC3339 format](https://www.rfc-editor.org/rfc/rfc3339).

Thank you!

```suggestion
                "type": "string",
                "format": "date-time"
```

File: v3.0-RC2/geofencing_zones.json [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+          },
+          "required": ["type", "features"]
+        },
+        "global_rules": {
+          "description": "Array of Rule objects defining restrictions that apply globally in all areas as the default restrictions, except where overridden with an explicit geofencing zone.",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "vehicle_type_id": {
```
Comment (richfab) [inline v3.0-RC2/geofencing_zones.json:163]: Let's please replace `vehicle_type_id` with `vehicle_type_ids` here as well. Thank you!

```suggestion
              "vehicle_type_ids": {
```

File: v3.0-RC2/station_information.json [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+              },
+              "vehicle_docks_capacity": {
+                "description": "This field's value is an array of objects containing the keys vehicle_type_id and count defined below. These objects are used to model the total docking capacity of a station, both available and unavailable, for each type of vehicle defined in vehicle_types.json.",
+                "type": "array",
+                "items": {
+                  "type": "object",
+                  "properties": {
+                    "vehicle_type_ids": {
+                      "description": "A vehicle_type_id, as defined in vehicle_types.json, that may dock at the station.",
+                      "type": "string"
```
Comment (richfab) [inline v3.0-RC2/station_information.json:221]: Let's please change the type of `vehicle_type_ids` to an array of strings as defined in https://github.com/MobilityData/gbfs/pull/547.

Thank you!

```suggestion
                      "type": "array",
                      "items": {
                        "type": "string"
                      }
```

File: v3.0-RC2/station_information.json [hunk 5]
```diff
@@ -0,0 +1,276 @@
...
+                    },
+                    "count": {
+                      "description": "The total number of docks at the station, both available and unavailable, that may accept the corresponding vehicle type as defined by its vehicle_type_id.",
+                      "type": "integer",
+                      "minimum": 0
+                    }
+                  },
+                  "required": ["vehicle_type_id","count"]
```
Comment (richfab) [inline v3.0-RC2/station_information.json:229]: Typo

```suggestion
                  "required": ["vehicle_type_ids","count"]
```

File: v3.0-RC2/station_information.json [hunk 6]
```diff
@@ -0,0 +1,276 @@
...
+              },
+              "vehicle_types_capacity": {
+                "description": "This field's value is an array of objects containing the keys vehicle_type_ids and count defined below. These objects are used to model the parking capacity of virtual stations (defined using the is_virtual_station field) for each vehicle type that can be returned to this station.",
+                "type": "array",
+                "items": {
+                  "type": "object",
+                  "properties": {
+                    "vehicle_type_ids": {
+                      "description": "A vehicle_type_id, as defined in vehicle_types.json, that may park at the virtual station.",
+                      "type": "string"
```
Comment (richfab) [inline v3.0-RC2/station_information.json:202]: Let's please change the type of `vehicle_type_ids` to an array of strings as defined in https://github.com/MobilityData/gbfs/pull/547.

Thank you!

```suggestion
                      "type": "array",
                      "items": {
                        "type": "string"
                      }
```

**Review by richfab — CHANGES_REQUESTED** [review]
Awesome contribution @tdelmas! Thank you very much!

I added a few change requests.

Also, could you please add `"3.0-RC2"` to the file `v3.0-RC/gbfs_versions.json` like for the other schemas? Thank you!

File: v3.0-RC2/station_information.json [hunk 7]
```diff
@@ -0,0 +1,276 @@
...
+                "type": "array",
+                "items": {
+                  "type": "object",
+                  "properties": {
+                    "vehicle_type_ids": {
+                      "description": "A vehicle_type_id, as defined in vehicle_types.json, that may dock at the station.",
+                      "type": "string"
+                    },
+                    "count": {
+                      "description": "The total number of docks at the station, both available and unavailable, that may accept the corresponding vehicle type as defined by its vehicle_type_id.",
```
Comment (richfab) [inline v3.0-RC2/station_information.json:224]: Let's please update the description to match the [spec](https://github.com/MobilityData/gbfs/blob/v3.0-RC2/gbfs.md#station_informationjson). Thank you!

```suggestion
                      "description": "A number representing the total number of docks at the station, both available and unavailable, that may accept the vehicle types specified by vehicle_type_ids.",
```

File: v3.0-RC2/station_information.json [hunk 8]
```diff
@@ -0,0 +1,276 @@
...
+                }
+              },
+              "vehicle_docks_capacity": {
+                "description": "This field's value is an array of objects containing the keys vehicle_type_id and count defined below. These objects are used to model the total docking capacity of a station, both available and unavailable, for each type of vehicle defined in vehicle_types.json.",
+                "type": "array",
+                "items": {
+                  "type": "object",
+                  "properties": {
+                    "vehicle_type_ids": {
+                      "description": "A vehicle_type_id, as defined in vehicle_types.json, that may dock at the station.",
```
Comment (richfab) [inline v3.0-RC2/station_information.json:220]: Let's please update the description to match the [spec](https://github.com/MobilityData/gbfs/blob/v3.0-RC2/gbfs.md#station_informationjson). Thank you!

```suggestion
                      "description": "An array of strings where each string represents a vehicle_type_id that is able to use a particular type of dock at the station.",
```

File: v3.0-RC2/station_information.json [hunk 9]
```diff
@@ -0,0 +1,276 @@
...
+                "minimum": 0
+              },
+              "vehicle_types_capacity": {
+                "description": "This field's value is an array of objects containing the keys vehicle_type_ids and count defined below. These objects are used to model the parking capacity of virtual stations (defined using the is_virtual_station field) for each vehicle type that can be returned to this station.",
+                "type": "array",
+                "items": {
+                  "type": "object",
+                  "properties": {
+                    "vehicle_type_ids": {
+                      "description": "A vehicle_type_id, as defined in vehicle_types.json, that may park at the virtual station.",
```
Comment (richfab) [inline v3.0-RC2/station_information.json:201]: Let's please update the description to match the [spec](https://github.com/MobilityData/gbfs/blob/v3.0-RC2/gbfs.md#station_informationjson). Thank you!

```suggestion
                      "description": "The vehicle_type_ids, as defined in vehicle_types.json, that may park at the virtual station.",
```

File: v3.0-RC2/station_information.json [hunk 10]
```diff
@@ -0,0 +1,276 @@
...
+                      "description": "A number representing the total number of vehicles of the corresponding vehicle_type_id type that can park within the virtual station.",
+                      "type": "integer",
+                      "minimum": 0
+                    }
+                  },
+                  "required": ["vehicle_type_id","count"]
+                }
+              },
+              "vehicle_docks_capacity": {
+                "description": "This field's value is an array of objects containing the keys vehicle_type_id and count defined below. These objects are used to model the total docking capacity of a station, both available and unavailable, for each type of vehicle defined in vehicle_types.json.",
```
Comment (richfab) [inline v3.0-RC2/station_information.json:217]: Let's please update the description to match the [spec](https://github.com/MobilityData/gbfs/blob/v3.0-RC2/gbfs.md#station_informationjson). Thank you!

```suggestion
                "description": "This field's value is an array of objects containing the keys vehicle_type_ids and count defined below. These objects are used to model the total docking capacity of a station, both available and unavailable, for each type of vehicle that may dock at this station.",
```

File: v3.0-RC2/station_information.json [hunk 11]
```diff
@@ -0,0 +1,276 @@
...
+                "type": "array",
+                "items": {
+                  "type": "object",
+                  "properties": {
+                    "vehicle_type_ids": {
+                      "description": "A vehicle_type_id, as defined in vehicle_types.json, that may park at the virtual station.",
+                      "type": "string"
+                    },
+                    "count": {
+                      "description": "A number representing the total number of vehicles of the corresponding vehicle_type_id type that can park within the virtual station.",
```
Comment (richfab) [inline v3.0-RC2/station_information.json:208]: Let's please update the description to match the [spec](https://github.com/MobilityData/gbfs/blob/v3.0-RC2/gbfs.md#station_informationjson). Thank you!

```suggestion
                      "description": "A number representing the total number of vehicles of the specified vehicle_type_ids that can park within the virtual station.",
```

File: v3.0-RC/gbfs_versions.json
Comment (richfab) [inline v3.0-RC/gbfs_versions.json:1]: Actually, in this file I meant to add `"3.0-RC2"` to `data.versions` enum, not to `version` enum. 

Can you please add `"3.0-RC2"` to `data.versions` enum and remove it from `version` enum?

Sorry for the confusion. Thank you!

Comment (tdelmas) [reply v3.0-RC/gbfs_versions.json:1]: Done, in the two missing places

**Comment by tdelmas** [comment]
@richfab thank you for that very detailed review! Everything should be fix now

File: v3.0-RC2/gbfs_versions.json [hunk 12]
```diff
@@ -0,0 +1,62 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://github.com/NABSA/gbfs/blob/v3.0-RC2/gbfs.md#gbfs_versionsjson-added-in-v11",
+  "description": "Lists all feed endpoints published according to version sof the GBFS documentation. (added in v1.1)",
+  "type": "object",
+  "properties": {
+    "last_updated": {
+      "description": "Last time the data in the feed was updated in POSIX time.",
```
Comment (richfab) [inline v3.0-RC2/gbfs_versions.json:8]:
```suggestion
      "description": "Last time the data in the feed was updated in RFC3339 format.",
```

File: v3.0-RC2/geofencing_zones.json [hunk 13]
```diff
@@ -0,0 +1,202 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://github.com/NABSA/gbfs/blob/v3.0-RC2/gbfs.md#geofencing_zonesjson",
+  "description": "Describes geofencing zones and their associated rules and attributes (added in v2.1-RC).",
+  "type": "object",
+  "properties": {
+    "last_updated": {
+      "description": "Last time the data in the feed was updated in POSIX time.",
```
Comment (richfab) [inline v3.0-RC2/geofencing_zones.json:8]:
```suggestion
      "description": "Last time the data in the feed was updated in RFC3339 format.",
```

File: v3.0-RC2/manifest.json [hunk 14]
```diff
@@ -0,0 +1,86 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://github.com/NABSA/gbfs/blob/v3.0-RC2/gbfs.md#manifestjson",
+  "description": "An index of gbfs.json URLs for each GBFS data set produced by a publisher. A single instance of this file should be published at a single stable URL, for example: https://example.com/gbfs/manifest.json.",
+  "type": "object",
+  "properties": {
+    "last_updated": {
+      "description": "Last time the data in the feed was updated in POSIX time.",
```
Comment (richfab) [inline v3.0-RC2/manifest.json:8]:
```suggestion
      "description": "Last time the data in the feed was updated in RFC3339 format.",
```

File: v3.0-RC2/station_information.json [hunk 15]
```diff
@@ -0,0 +1,282 @@
...
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id":
+    "https://github.com/NABSA/gbfs/blob/v3.0-RC2/gbfs.md#station_informationjson",
+  "description":
+    "List of all stations, their capacities and locations. REQUIRED of systems utilizing docks.",
+  "type": "object",
+  "properties": {
+    "last_updated": {
+      "description":
+        "Last time the data in the feed was updated in POSIX time.",
```
Comment (richfab) [inline v3.0-RC2/station_information.json:11]:
```suggestion
        "Last time the data in the feed was updated in RFC3339 format.",
```

File: v3.0-RC2/station_status.json [hunk 16]
```diff
@@ -0,0 +1,149 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://github.com/NABSA/gbfs/blob/v3.0-RC2/gbfs.md#station_statusjson",
+  "description":
+    "Describes the capacity and rental availability of the station",
+  "type": "object",
+  "properties": {
+    "last_updated": {
+      "description":
+        "Last time the data in the feed was updated in POSIX time.",
```
Comment (richfab) [inline v3.0-RC2/station_status.json:10]:
```suggestion
        "Last time the data in the feed was updated in RFC3339 format.",
```

File: v3.0-RC2/system_alerts.json [hunk 17]
```diff
@@ -0,0 +1,165 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://github.com/NABSA/gbfs/blob/v3.0-RC2/gbfs.md#system_alertsjson",
+  "description": "Describes ad-hoc changes to the system.",
+  "type": "object",
+  "properties": {
+    "last_updated": {
+      "description":
+        "Last time the data in the feed was updated in POSIX time.",
```
Comment (richfab) [inline v3.0-RC2/system_alerts.json:9]:
```suggestion
        "Last time the data in the feed was updated in RFC3339 format.",
```

File: v3.0-RC2/system_information.json [hunk 18]
```diff
@@ -0,0 +1,1235 @@
...
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id":
+    "https://github.com/NABSA/gbfs/blob/v3.0-RC2/gbfs.md#system_informationjson",
+  "description":
+    "Details including system operator, system location, year implemented, URL, contact info, time zone.",
+  "type": "object",
+  "properties": {
+    "last_updated": {
+      "description":
+        "Last time the data in the feed was updated in POSIX time.",
```
Comment (richfab) [inline v3.0-RC2/system_information.json:11]:
```suggestion
        "Last time the data in the feed was updated in RFC3339 format.",
```

File: v3.0-RC2/system_pricing_plans.json [hunk 19]
```diff
@@ -0,0 +1,205 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id":
+    "https://github.com/NABSA/gbfs/blob/v3.0-RC2/gbfs.md#system_pricing_plansjson",
+  "description": "Describes the pricing schemes of the system.",
+  "type": "object",
+  "properties": {
+    "last_updated": {
+      "description":
+        "Last time the data in the feed was updated in POSIX time.",
```
Comment (richfab) [inline v3.0-RC2/system_pricing_plans.json:10]:
```suggestion
        "Last time the data in the feed was updated in RFC3339 format.",
```

File: v3.0-RC2/system_regions.json [hunk 20]
```diff
@@ -0,0 +1,72 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://github.com/NABSA/gbfs/blob/v3.0-RC2/gbfs.md#system_regionsjson",
+  "description":
+    "Describes regions for a system that is broken up by geographic or political region.",
+  "type": "object",
+  "properties": {
+    "last_updated": {
+      "description":
+        "Last time the data in the feed was updated in POSIX time.",
```
Comment (richfab) [inline v3.0-RC2/system_regions.json:10]:
```suggestion
        "Last time the data in the feed was updated in RFC3339 format.",
```

File: v3.0-RC2/vehicle_status.json [hunk 21]
```diff
@@ -0,0 +1,156 @@
...
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id":
+    "https://github.com/NABSA/gbfs/blob/v3.0-RC2/gbfs.md#vehicle_statusjson",
+  "description":
+    "Describes the vehicles that are available for rent (as of v3.0-RC, formerly free_bike_status).",
+  "type": "object",
+  "properties": {
+    "last_updated": {
+      "description":
+        "Last time the data in the feed was updated in POSIX time.",
```
Comment (richfab) [inline v3.0-RC2/vehicle_status.json:11]:
```suggestion
        "Last time the data in the feed was updated in RFC3339 format.",
```

File: v3.0-RC2/vehicle_types.json [hunk 22]
```diff
@@ -0,0 +1,272 @@
...
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id":
+    "https://github.com/NABSA/gbfs/blob/v3.0-RC2/gbfs.md#vehicle_typesjson-added-in-v21-rc",
+  "description":
+    "Describes the types of vehicles that System operator has available for rent (added in v2.1-RC).",
+  "type": "object",
+  "properties": {
+    "last_updated": {
+      "description":
+        "Last time the data in the feed was updated in POSIX time.",
```
Comment (richfab) [inline v3.0-RC2/vehicle_types.json:11]:
```suggestion
        "Last time the data in the feed was updated in RFC3339 format.",
```

File: v3.0-RC2/station_information.json [hunk 23]
```diff
@@ -0,0 +1,282 @@
...
+                        "type": "string"
+                      }
+                    },
+                    "count": {
+                      "description": "A number representing the total number of vehicles of the specified vehicle_type_ids that can park within the virtual station.",
+                      "type": "integer",
+                      "minimum": 0
+                    }
+                  },
+                  "required": ["vehicle_type_id","count"]
```
Comment (richfab) [inline v3.0-RC2/station_information.json:213]: Typo

```suggestion
                  "required": ["vehicle_type_ids","count"]
```

**Review by richfab — CHANGES_REQUESTED** [review]
Hi @tdelmas! Thank you very much for the fast changes ⚡ We can almost merge 🙌 

- The Timestamp needs to be changed to RFC3339 format in a few more descriptions. 
- `vehicle_type_ids` needs to be changed to plural in one more place.

Thanks to you the JSON Schema are up to date very quickly, so thank you! 🙏

**Review by richfab — APPROVED** [review]
LGTM! 👏 
Merging now 🚀 
Thank you!

**Pull request merged**